{
  "uid" : "ead8f5bb18280980",
  "name" : "test__registration",
  "fullName" : "test.conduit_panniangyan_test.TestConduit#test__registration",
  "historyId" : "968ef86fef284f9344208081067f8ec4",
  "time" : {
    "start" : 1627898435312,
    "stop" : 1627898435312,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "TypeError: setup() takes 0 positional arguments but 1 was given",
  "statusTrace" : "cls = <class '_pytest.runner.CallInfo'>\nfunc = <function call_runtest_hook.<locals>.<lambda> at 0x7fa5fe192700>\nwhen = 'setup'\nreraise = (<class '_pytest.outcomes.Exit'>, <class 'KeyboardInterrupt'>)\n\n    @classmethod\n    def from_call(\n        cls,\n        func: \"Callable[[], TResult]\",\n        when: \"Literal['collect', 'setup', 'call', 'teardown']\",\n        reraise: Optional[\n            Union[Type[BaseException], Tuple[Type[BaseException], ...]]\n        ] = None,\n    ) -> \"CallInfo[TResult]\":\n        excinfo = None\n        start = timing.time()\n        precise_start = timing.perf_counter()\n        try:\n>           result: Optional[TResult] = func()\n\n/opt/hostedtoolcache/Python/3.9.5/x64/lib/python3.9/site-packages/_pytest/runner.py:311: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.9.5/x64/lib/python3.9/site-packages/_pytest/runner.py:255: in <lambda>\n    lambda: ihook(item=item, **kwds), when=when, reraise=reraise\n/opt/hostedtoolcache/Python/3.9.5/x64/lib/python3.9/site-packages/pluggy/hooks.py:286: in __call__\n    return self._hookexec(self, self.get_hookimpls(), kwargs)\n/opt/hostedtoolcache/Python/3.9.5/x64/lib/python3.9/site-packages/pluggy/manager.py:93: in _hookexec\n    return self._inner_hookexec(hook, methods, kwargs)\n/opt/hostedtoolcache/Python/3.9.5/x64/lib/python3.9/site-packages/pluggy/manager.py:84: in <lambda>\n    self._inner_hookexec = lambda hook, methods, kwargs: hook.multicall(\n/opt/hostedtoolcache/Python/3.9.5/x64/lib/python3.9/site-packages/_pytest/nose.py:13: in pytest_runtest_setup\n    call_optional(item.parent.obj, \"setup\")\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nobj = <conduit_panniangyan_test.TestConduit object at 0x7fa5fe0dbee0>\nname = 'setup'\n\n    def call_optional(obj, name):\n        method = getattr(obj, name, None)\n        isfixture = hasattr(method, \"_pytestfixturefunction\")\n        if method is not None and not isfixture and callable(method):\n            # If there's any problems allow the exception to raise rather than\n            # silently ignoring them.\n>           method()\nE           TypeError: setup() takes 0 positional arguments but 1 was given\n\n/opt/hostedtoolcache/Python/3.9.5/x64/lib/python3.9/site-packages/_pytest/nose.py:38: TypeError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "parentSuite",
    "value" : "test"
  }, {
    "name" : "suite",
    "value" : "conduit_panniangyan_test"
  }, {
    "name" : "subSuite",
    "value" : "TestConduit"
  }, {
    "name" : "host",
    "value" : "fv-az74-587"
  }, {
    "name" : "thread",
    "value" : "3661-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "test.conduit_panniangyan_test"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 15,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 15
      },
      "items" : [ {
        "uid" : "307944bc46b412e2",
        "reportUrl" : "https://panniangyan.github.io/conduit/17//#testresult/307944bc46b412e2",
        "status" : "broken",
        "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=52653): Max retries exceeded with url: /session/f04a3cdd168c74da301d0c38b9cb3db9/url (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fca1c98d880>: Failed to establish a new connection: [Errno 111] Connection refused'))",
        "time" : {
          "start" : 1627896378518,
          "stop" : 1627896378518,
          "duration" : 0
        }
      }, {
        "uid" : "427644a94ef45b1b",
        "reportUrl" : "https://panniangyan.github.io/conduit/16//#testresult/427644a94ef45b1b",
        "status" : "broken",
        "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=60017): Max retries exceeded with url: /session/b1a5c94c7bc7022ef76dd2ebb8ba0f70/url (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1a75aa40d0>: Failed to establish a new connection: [Errno 111] Connection refused'))",
        "time" : {
          "start" : 1627895212095,
          "stop" : 1627895212095,
          "duration" : 0
        }
      }, {
        "uid" : "7edf29454996e51a",
        "reportUrl" : "https://panniangyan.github.io/conduit/15//#testresult/7edf29454996e51a",
        "status" : "broken",
        "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=43719): Max retries exceeded with url: /session/91742e4ead70df7aa6cbcad06f4f1bd2/url (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe18db6bc70>: Failed to establish a new connection: [Errno 111] Connection refused'))",
        "time" : {
          "start" : 1627892995669,
          "stop" : 1627892995669,
          "duration" : 0
        }
      }, {
        "uid" : "4fbb63d7cd536e66",
        "reportUrl" : "https://panniangyan.github.io/conduit/14//#testresult/4fbb63d7cd536e66",
        "status" : "broken",
        "statusDetails" : "AttributeError: 'WebDriver' object has no attribute 'webdriver'",
        "time" : {
          "start" : 1627892617354,
          "stop" : 1627892617354,
          "duration" : 0
        }
      }, {
        "uid" : "1e6c2da72bf817cd",
        "reportUrl" : "https://panniangyan.github.io/conduit/12//#testresult/1e6c2da72bf817cd",
        "status" : "broken",
        "statusDetails" : "AttributeError: 'TestConduit' object has no attribute 'browser'",
        "time" : {
          "start" : 1627888455618,
          "stop" : 1627888455618,
          "duration" : 0
        }
      }, {
        "uid" : "8583a58aa9210346",
        "reportUrl" : "https://panniangyan.github.io/conduit/11//#testresult/8583a58aa9210346",
        "status" : "broken",
        "statusDetails" : "AttributeError: 'TestConduit' object has no attribute 'browser'",
        "time" : {
          "start" : 1627829932546,
          "stop" : 1627829932546,
          "duration" : 0
        }
      }, {
        "uid" : "c59acee91c6e3928",
        "reportUrl" : "https://panniangyan.github.io/conduit/10//#testresult/c59acee91c6e3928",
        "status" : "broken",
        "statusDetails" : "AttributeError: 'TestConduit' object has no attribute 'browser'",
        "time" : {
          "start" : 1627829830025,
          "stop" : 1627829830025,
          "duration" : 0
        }
      }, {
        "uid" : "2d3c931a1cfbf27e",
        "reportUrl" : "https://panniangyan.github.io/conduit/9//#testresult/2d3c931a1cfbf27e",
        "status" : "broken",
        "statusDetails" : "AttributeError: 'TestConduit' object has no attribute 'browser'",
        "time" : {
          "start" : 1627829049972,
          "stop" : 1627829049972,
          "duration" : 0
        }
      }, {
        "uid" : "db670bd415a60c20",
        "reportUrl" : "https://panniangyan.github.io/conduit/8//#testresult/db670bd415a60c20",
        "status" : "broken",
        "statusDetails" : "AttributeError: 'TestConduit' object has no attribute 'browser'",
        "time" : {
          "start" : 1627828717718,
          "stop" : 1627828717718,
          "duration" : 0
        }
      }, {
        "uid" : "4ccb4a24db3d96df",
        "reportUrl" : "https://panniangyan.github.io/conduit/7//#testresult/4ccb4a24db3d96df",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)",
        "time" : {
          "start" : 1627828252518,
          "stop" : 1627828252518,
          "duration" : 0
        }
      }, {
        "uid" : "8b9482fd3ef6d0ca",
        "reportUrl" : "https://panniangyan.github.io/conduit/6//#testresult/8b9482fd3ef6d0ca",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)",
        "time" : {
          "start" : 1627827065771,
          "stop" : 1627827065771,
          "duration" : 0
        }
      }, {
        "uid" : "beb59897c53c98fb",
        "reportUrl" : "https://panniangyan.github.io/conduit/5//#testresult/beb59897c53c98fb",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)",
        "time" : {
          "start" : 1627819953717,
          "stop" : 1627819953717,
          "duration" : 0
        }
      }, {
        "uid" : "82efe754d9ec88e",
        "reportUrl" : "https://panniangyan.github.io/conduit/4//#testresult/82efe754d9ec88e",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)",
        "time" : {
          "start" : 1627818407981,
          "stop" : 1627818407981,
          "duration" : 0
        }
      }, {
        "uid" : "3ef007e12fb7c524",
        "reportUrl" : "https://panniangyan.github.io/conduit/3//#testresult/3ef007e12fb7c524",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)",
        "time" : {
          "start" : 1627818158682,
          "stop" : 1627818158682,
          "duration" : 0
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "ead8f5bb18280980.json",
  "parameterValues" : [ ]
}